src/hooks/use-authentication/types.ts

```diff
@@ -19,7 +24,7 @@ interface UseAuthenticationOptions {
 
 interface UseAuthenticationResult {
 	// https://github.com/nextauthjs/next-auth/blob/next-auth@v4.10.3/packages/next-auth/src/core/types.ts#L422-L440
-	error: undefined | Session['error']
+	error: SessionData['error']
```

Comment:
Apparently 'error' isn't on the next auth session type anymore, same with `status`